backend/sim/wire/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 - See NOTICE file for copyright holders.
```

Comment:
Why do you think it should be done? It's not necessary, maybe even disregarded.
https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

Eventually we could move to SPDX license identifiers which don't include a year.

Reply:
Well because I thought it's our standard to update the copyright year of a source file when we're editing this file and I could this feedback in one of my PRs where I forgot to do so.